src/root/views/region.js

```diff
@@ -245,21 +252,32 @@ class AdminArea extends SFPComponent {
                         <KeyFiguresHeader fullscreen={this.state.fullscreen} appealsListStats={this.props.appealsListStats} />
                       ) : null}
                       <div className={c('inner', {'appeals--fullscreen': this.state.fullscreen})}>
-                        <AppealsTable
-                          title={strings.regionAppealsTableTitle}
-                          region={getRegionId(this.props.match.params.id)}
-                          regionOperations={this.props.appealStats}
-                          mapBoundingBox={mapBoundingBox}
-                          mapLayers={[this.state.maskLayer]}
```

Comment:
Not sure what this did, but I can't find it being used or passed down anywhere. Tried to pass it to the `<MainMap>` component as `layers` prop, but it gave a couple of errors.

@frozenhelium @batpad